storage/api/Storage/Storage.cs

```diff
@@ -1024,6 +1222,62 @@ public int Run(string[] args)
                     case "get-requester-pays":
                         if (args.Length < 2 && PrintUsage()) return -1;
                         return GetRequesterPays(args[1]) ? 1 : 0;
+                        break;

```

Comment:
Remove break b/c the return statement above it will prevent this from being executed.